get-song-name.js

```diff
@@ -12,13 +12,26 @@ if(document.querySelectorAll('.trackView').length>0){
       scrapedSong.songNumber = songNumber;
     }
   });
+
```

Comment:
Extract the query selector below into a variable out here and then use it as part of the if check to avoid an `innerHTML of undefined error` in some random edge case we might not know about.